lib/fs.js

```diff
@@ -1102,7 +1102,7 @@ function futimesSync(fd, atime, mtime) {
 
 function writeAll(fd, isUserFd, buffer, offset, length, position, callback) {
   // write(fd, buffer, offset, length, position, callback)
-  fs.write(fd, buffer, offset, length, position, function(writeErr, written) {
+  fs.write(fd, buffer, offset, length, position, function writeCB(writeErr, written) {
```

Comment:
This line exceeds the maximum line length. It could be written as:

```js
fs.write(fd, buffer, offset, length, position,
             function writeCB(writeErr, written) {
```

Reply:
@laertis90 You can follow [Step 9](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#step-9-discuss-and-update) to edit and update the commit

Reply:
cc @laertis90